app/assets/stylesheets/style.scss

```diff
@@ -23,10 +23,14 @@ body {
 
 .course-info {
   font-size: 80%;
+}
+
+.instructor-info {
+  font-size: 80%;
   padding-left: 15px;
 }
 
-.course-info span img {
+.instructor-info span img {
```

Comment:
Merge rule `.instructor-info span img` with rule on line 28<br>Selector should have depth of applicability no greater than 2, but was 3